src/cls/tabular/cls_tabular.cc

```diff
@@ -56,6 +56,39 @@ static std::string string_ncopy(const char* buffer, std::size_t buffer_size) {
   return std::string(buffer, copyupto);
 }
 
+// Get fb_seq_num from xattr
+static
+int get_fb_seq_num(cls_method_context_t hctx, int& fb_seq_num) {
+    
+    bufferlist fb_bl2;
+    int r1 = cls_cxx_getxattr(hctx, "fb_seq_num", &fb_bl2);
+    if (r1 == -ENOENT || r1 == -ENODATA) {
+        fb_seq_num = Tables::FB_SEQ_NUM_MIN;
+        // If fb_seq_num is not present then insert it in xattr. 
+    }
+    else if( r1 < 0 ) {
+        return r1;
+    }
+    else {
+        bufferlist::iterator it = fb_bl2.begin();
+        ::decode(fb_seq_num,it);
+    }
+    return 0;
+
+}
+
+// Update counter and Insert fb_seq_num to xattr
+static
+int set_fb_seq_num(cls_method_context_t hctx, int fb_seq_num) {
+
+    bufferlist fb_bl2;
+    fb_seq_num++;
```

Comment:
Let's not increment here.  Assume the value passed in is the current num to store.